testing/py-filelock/APKBUILD

```diff
@@ -0,0 +1,53 @@
+# Contributor: Dmitry Romanenko <dmitry@romanenko.in>
+# Maintainer: Dmitry Romanenko <dmitry@romanenko.in>
+pkgname=py-filelock
+_pkgname=filelock
+pkgver=3.0.10
+pkgrel=0
+pkgdesc="A platform independent file lock for Python"
+url="https://github.com/benediktschmitt/py-filelock"
+arch="noarch"
+license="Unlicense"
+depends=""
+checkdepends="pytest"
+makedepends="python2-dev python3-dev py-setuptools"
+subpackages="py3-$_pkgname:_py3 py2-$_pkgname:_py2"
+source="$pkgname-$pkgver.tar.gz::https://github.com/benediktschmitt/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir"/$pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
+}
+
+check() {
+	cd "$builddir"
+	python2 -m pytest -xvv test.py
+	python3 -m pytest -xvv test.py
+}
+
+package() {
+	mkdir -p "$pkgdir"
+}
+
+_py2() {
+	replaces="$pkgname"
```

Comment:
Not needed as it's a new aport and there was never a `py-filelock` package.

Reply:
Tho, for longterm simple upgrades would it give some benefit?